tests/console/docker_image.pm

```diff
@@ -7,69 +7,70 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Test installation and running of the docker image for this snapshot
-# Maintainer: Fabian Vogt <fvogt@suse.com>
+# Summary: Test installation and running of the docker image from the registry for this snapshot
+# Maintainer: Pavel Dostál <pdostal@suse.cz>
 
 use base 'consoletest';
 use testapi;
 use utils;
 use strict;
-use version_utils qw(is_leap is_tumbleweed);
+use registration;
+use version_utils;
```

Comment:
I would do the same here as you did in docker_image_rpm.pm below, specify what you use from registration and version_utils
`use version_utils qw(is_leap is_tumbleweed);`